README.md

```diff
@@ -101,6 +101,10 @@ meson _build --prefix=/usr/local
 ninja -C _build install
 ```
 
+## Link to the guide to enable the podman socket
```

Comment:
2 Things:

1. The heading should be briefer, like "Podman Socket Activation"
2. As every heading has an emoji, I suggest to use 🔌 emoji here.